Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -452,6 +454,8 @@ static bool isPseudoClassValidAfterPseudoElement(CSSSelector::PseudoClass pseudo
         return pseudoClass == CSSSelector::PseudoClass::WindowInactive;
     case CSSSelector::PseudoElement::UserAgentPart:
     case CSSSelector::PseudoElement::UserAgentPartLegacyAlias:
+    // FIXME: Should WebKitUnknown be included here?
+    case CSSSelector::PseudoElement::WebKitUnknown:
```

Comment:
We should make this use the `default:` case, chaining an unknown webkit pseudo element and a pseudo class should not be allowed.

Reply:
I tend to agree, but that would be a change in behavior: http://software.hixie.ch/utilities/js/live-dom-viewer/?%3C!DOCTYPE%20html%3E%0A%3Cstyle%3E%3A%3A-webkit-asdf%3Afocus%2C%20body%20%7B%20background%3Ablue%20%7D%20%3C%2Fstyle%3E

I'd rather make that change separately from the @supports case, with its own tests.

Reply:
Seems fine to remove as a followup.